wd_core.au3

```diff
@@ -172,6 +173,7 @@ Global $_WD_DRIVER_CLOSE = True ; Close prior driver instances before launching
 Global $_WD_DRIVER_DETECT = True ; Don't launch new driver instance if one already exists
 Global $_WD_RESPONSE_TRIM = 100 ; Trim response string to given value for debug output
 Global $_WD_ERROR_MSGBOX = True ; Shows in compiled scripts error messages in msgboxes
+Global $_WD_ERROR_OUTPUTDEBUG = False ; Log errors to "OutputDebugString"

```

Comment:
`OutputDebugString` is disabled by default. So is it `ScriptBreakingChange` ? or only requires Warrning in relase note ?

Reply:
No, I wouldn't consider that script breaking.